### ISSUE TITLE
parseBbCode Cannot read property 'replace' of null

### ISSUE THREAD

**Issue by xPaw** [DoctorMcKay/node-steam-user#222]
```
node_modules/steam-user/components/chatroom.js:571
        str = str.replace(/\\\[/g, replacement);
                  ^

TypeError: Cannot read property 'replace' of null
    at parseBbCode (node_modules/steam-user/components/chatroom.js:571:12)
    at SteamUser.<anonymous> (node_modules/steam-user/components/chatroom.js:66:32)
    at handlers.forEach (node_modules/steam-user/components/classes/HandlerManager.js:37:12)
    at Array.forEach (<anonymous>)
    at HandlerManager.emit (node_modules/steam-user/components/classes/HandlerManager.js:36:12)
    at SteamUser._handleMessage (node_modules/steam-user/components/messages.js:544:24)
    at SteamUser._handleNetMessage (node_modules/steam-user/components/messages.js:470:7)
    at SteamUser.processMulti (node_modules/steam-user/components/messages.js:571:9)
    at SteamUser.<anonymous> (node_modules/steam-user/components/messages.js:565:16)
    at handlers.forEach (node_modules/steam-user/components/classes/HandlerManager.js:37:12)
```

I don't know what message caused it though.

**Comment by DoctorMcKay** [comment]
Looks like it was an incoming chat-room message, but `message` was null. I would guess a EChatEntryType other than Chat? I didn't account for that. A EChatEntryType of Typing in an incoming friend message has `message` as an empty string, but maybe there's some other type that sends null instead..

**Comment by DoctorMcKay** [comment]
Fixed in [v4.2.1](https://github.com/DoctorMcKay/node-steam-user/releases/tag/v4.2.1).
